src/main/java/org/mskcc/cbio/portal/dao/DaoGeneticAlteration.java

```diff
@@ -32,25 +32,39 @@
 
 package org.mskcc.cbio.portal.dao;
 
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.apache.commons.lang3.StringUtils;
 import org.mskcc.cbio.portal.model.CanonicalGene;
+import org.mskcc.cbio.portal.model.GeneticAlterationType;
+import org.mskcc.cbio.portal.model.GeneticProfile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
 import java.util.Map.Entry;
-
-import com.fasterxml.jackson.databind.node.ObjectNode;
-
-import org.apache.commons.lang3.StringUtils;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
  * Data Access Object for the Genetic Alteration Table.
  *
  * @author Ethan Cerami.
  */
 public class DaoGeneticAlteration {
+
+    private static final Logger log = LoggerFactory.getLogger(DaoGeneticAlteration.class);
```

Comment:
logger not used. Did some code get lost by mistake?

Reply:
Done https://github.com/cBioPortal/cbioportal-core/pull/46/commits/989604a63beeeca9ec030d4ef21f9e09428763c7